drivers/serial/Kconfig.nrfx

```diff
@@ -50,6 +50,13 @@ config UART_NRF_DK_SERIAL_WORKAROUND_WAIT_MS
 	default 7
 	help
 	  Busy wait time (in milliseconds).
+
+# Increase stack size in tests built with no optimizations, as with the above
+# workaround enabled, the standard size may be insufficient. Use 512 to align
```

Comment:
> as with the above workaround enabled

I really don't understand this sentence.
Does it mean that this setting is somehow depending on the above workaround being enabled, or what ?

I suggest removing this, and make it clear the relationship to `above workaround`.

Also please be more specific than `above workaround`, because some time from now someone could introduce a setting between `TEST_EXTRA_STACK_SIZE` and `UART_NRF_DK_SERIAL_WORKAROUND_WAIT_MS` and not updating this comment.

Reply:
Rephrased the comment.